app/views/layouts/mailer.html.erb

```diff
@@ -1,13 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Any reason why this file was removed? Does it have any effect on this feature? 🤔

Reply:
We didnit use it!  got it from devise i think!

Reply:
This file comes from the rails scaffold..

Reply:
still works with out it ;)

Reply:
Oops :p should we put it back?

Reply:
I guess it'll take a while before you learn not to touch things that aren't directly related to the feature you're working on huh? 🤔 🙂

Not a blocker really... Just trying to keep you guys in line

Reply:
I would put it back. Yeah, just put it back.